lib/routes/user.js

```diff
@@ -0,0 +1,33 @@
+import express from 'express';
+import UserController from '../controllers/user.controller';
+import UserValidator from '../validators/user.validator';
+
+const router = express.Router();
+const userController = new UserController();
+const userValidator = new UserValidator();
+
+router.param('user_id', (req, res, next, user_id) => {
+    userController.find(req, res, next, user_id);
+});
+
+router.get('/:user_id', (req, res) => {
+    userController.getUser(req, res);
+});
+
+router.get('/', (req, res) => {
+    userController.getUsers(req, res);
+});
+
+router.post('/', userValidator.validate(), (req, res) => {
```

Comment:
It will be better if route will look like this - `router.post('/', userValidator.validate(), userController.create || create())

Reply:
Done